README.rst

```diff
@@ -97,23 +97,24 @@ Support
 
 Find support here:
 
-* `Leap user community <https://support.dwavesys.com/hc/en-us/community/topics>`_
-  to converse with a large community of D-Wave users.
-* `Leap help center <https://support.dwavesys.com/hc/en-us>`_
-  to search the Leap knowledge base.
-* `SDK GitHub repo <https://github.com/dwavesystems/dwave-ocean-sdk/issues>`_ to
-  open issues or request features on the Ocean SDK or on any one of its
-  `packages <https://github.com/dwavesystems>`_.
+*   `Leap user community <https://support.dwavesys.com/hc/en-us/community/topics>`_
+    to converse with a large community of D-Wave users.
+*   `Leap help center <https://support.dwavesys.com/hc/en-us>`_
+    to search the Leap knowledge base.
+*   `SDK GitHub repo <https://github.com/dwavesystems/dwave-ocean-sdk/issues>`_
+    to open issues or request features on the Ocean SDK or on any one of its
+    `packages <https://github.com/dwavesystems>`_.
 
 Contributing
 ============
 
 Your contributions are welcome!
 
-Ocean's `contributing guide <https://docs.ocean.dwavesys.com/en/stable/contributing.html>`_
+Ocean's
+`contributing guide <https://docs.dwavequantum.com/en/latest/ocean/contribute.html>`_
```

Comment:
Why split this in two lines?

Reply:
[PEP8](https://peps.python.org/pep-0008/#maximum-line-length): "Limit all lines to a maximum of 79 characters."

Reply:
Pretty sure that applies to python code only, not restructured text :smiley:

Reply:
But sure, if it's rendered the same, it doesn't matter.

Reply:
"all" lines, dude, no footnotes or asterisks.

Reply:
On second look, I see this is not a title, but regular body text (I don't know why I thought it was a heading).

However, if you want to limit line length to 79 chars, you should split the second line as well - it's 85 chars currently :laughing: